source/includes/backups/_10-create.md.erb

```diff
@@ -1,20 +1,35 @@
-## Create a backup
+## Schedules backup asynchronously.
```

Comment:
For consistency, I think we should call this heading `Schedule Backup` (and without the `.` at the end)

Reply:
Comment incorporated.